src/EnergyPlus/HVACManager.cc

```diff
@@ -1878,44 +1814,26 @@ void SimSelectedEquipment(EnergyPlusData &state,
     // METHODOLOGY EMPLOYED:
     // Each flag is checked and the appropriate manager is then called.
 
-    // Using/Aliasing
-    using NonZoneEquipmentManager::ManageNonZoneEquipment;
-    using PlantManager::ManagePlantLoops;
-    using PlantUtilities::AnyPlantLoopSidesNeedSim;
-    using PlantUtilities::ResetAllPlantInterConnectFlags;
-    using PlantUtilities::SetAllFlowLocks;
-    using SimAirServingZones::ManageAirLoops;
-    using ZoneEquipmentManager::ManageZoneEquipment;
-
-    // Locals
-    // SUBROUTINE ARGUMENT DEFINITIONS:
-    bool ResimulateAirZone; // True when solution technique on third iteration used in AirflowNetwork
-
     // SUBROUTINE PARAMETER DEFINITIONS:
     int constexpr MaxAir(5); // Iteration Max for Air Simulation Iterations
 
-    // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-    int IterAir; // counts iterations to enforce maximum iteration limit
-
-    IterAir = 0;
-
```

Comment:
[src/EnergyPlus/HVACManager.cc:1892]:(style),[variableScope],The scope of the variable 'ResimulateAirZone' can be reduced.
[src/EnergyPlus/HVACManager.cc:1898]:(style),[variableScope],The scope of the variable 'IterAir' can be reduced.